### PR TITLE
Updates changes.md file.

### PR DESCRIPTION
**Pull request by thibaultzanini**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,129 @@
+### v0.15.1
+
+-  **TreeController**
+
+   Updates to the new version.
+
+-  **Component**
+
+   Makes creating Component’s templateObjects lazy and on-demand.
```
Comment (dharcourt) [inline CHANGES.md]: Makes creating Component templateObjects lazy and on-demand.

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,129 @@
...
+   Reduces memory consumption.
+
+-  **Core**
+
+   Fixes super infinite loop in observed getter/setters.
+
+-  **DocumentResources**
+
+   Fix a bug affecting Firefox that was allowing components to be entering the document before its CSS stylesheets were loaded.
```
Comment (dharcourt) [inline CHANGES.md]: Fix a bug affecting Firefox that was allowing components to be entering the document before _their_ CSS stylesheets were loaded.

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,129 @@
...
+
+-  **EventManager**
+
+   Makes preparedForActivationEvents public.
+   Fixes a case where the first touch in an app would be lost.
+   Adds support of PointerEvents.
+   Adds support of devices with multiples pointer types. (mouse/touch).
+   PrepareForActivationEvents is called when a pointing device (mouse/touch) is moved over a component’s element.
+   Blocks emulated mouse events.
+   Adds an option `blocksEmulatedEvents` to disable the filtering of emulated mouse events.
```
Comment (dharcourt) [inline CHANGES.md]: Adds *a `blocksEmulatedEvents` option* to disable the filtering of emulated mouse events.

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,129 @@
...
+   Makes sure the PressComposer operate just when the main button is pressed.
+   Cancel a press when a scroll or a translate occur.
+
+-  **RangeController**
+
+   Enhance RangeController filter/sort binding speed.
+
+-  **Composer**
+
+   Composers load lazily their logic by default.
```
Comment (dharcourt) [inline CHANGES.md]: Composers *lazily load* their logic by default.

File: CHANGES.md [hunk 5]
```diff
@@ -1,3 +1,129 @@
...
+
+-  **Target**
+
+   Adds load/unload functions that will load composers.
+
+-  **TranslateComposer**
+
+   Support Pointer Events
+   Support devices with multiple pointer types.
+   Better stealing claimed pointers logic.
```
Comment (dharcourt) [inline CHANGES.md]: Better *logic to steal* claimed pointers.

File: CHANGES.md [hunk 6]
```diff
@@ -1,3 +1,129 @@
...
+   Fixes issues between composers that was claiming the same pointer.
```
Comment (dharcourt) [inline CHANGES.md]: Fixes issues *with* composers *claiming* the same pointer.

File: CHANGES.md [hunk 7]
```diff
@@ -1,3 +1,129 @@
...
+   Does not listener on wheel events by default.
```
Comment (dharcourt) [inline CHANGES.md]: Does not *listen* on wheel events by default.

File: CHANGES.md [hunk 8]
```diff
@@ -1,3 +1,129 @@
...
+   Fixes bug between inner scroller Components. (Was not able to work)
+   Add unload function. (release listeners)
```
Comment (dharcourt) [inline CHANGES.md]: Add unload function (releases listeners).

File: CHANGES.md [hunk 9]
```diff
@@ -1,3 +1,129 @@
...
+   Adds load/unload functions that will load composers.
+
+-  **TranslateComposer**
+
+   Support Pointer Events
+   Support devices with multiple pointer types.
+   Better stealing claimed pointers logic.
+   Fixes issues between composers that was claiming the same pointer.
+   Does not listener on wheel events by default.
+   Fixes bug between inner scroller Components. (Was not able to work)
```
Comment (dharcourt) [inline CHANGES.md]: Fixes bug *with* inner scroller Components (*were not working*).

File: CHANGES.md [hunk 10]
```diff
@@ -1,3 +1,129 @@
...
+   Add unload function. (release listeners)
+   Backwards compatibility for the Flow Translate Composer.
+   Adds supports for horizontal “wheel” movement.
+   Stops to raise translate events when a movement is beyond a extreme (max/min Translate X/Y).
```
Comment (dharcourt) [inline CHANGES.md]: Stops *sending* translate events when a movement is beyond a extreme (max/min Translate X/Y).

File: CHANGES.md [hunk 11]
```diff
@@ -1,3 +1,129 @@
...
+   Performance improvements. (less translateStart raised when handling wheel event, less listeners and diverse improvements)
```
Comment (dharcourt) [inline CHANGES.md]: Performance improvements (less translateStart *events sent* when handling wheel event, less listeners, and diverse improvements).

File: CHANGES.md [hunk 12]
```diff
@@ -1,3 +1,129 @@
...
+   Cancel a translate when a scroll occur.
```
Comment (dharcourt) [inline CHANGES.md]: Cancel a translate when a scroll *occurs*.

**Comment by thibaultzanini** [comment]
PR updated. Thanks @dharcourt

**Pull request merged**